contrib/cf-remote/cf_remote/main.py

```diff
@@ -54,6 +54,10 @@ def get_args():
     sp.add_argument("--edition", "-E", help="Enterprise or community packages", type=str)
     sp.add_argument("tags", metavar="TAG", nargs="*")
 
+    sp = subp.add_parser("list", help="List CFEngine packages available for download")
+    sp.add_argument("--edition", "-E", help="Enterprise or community packages", type=str)
```

Comment:
There's something like `choices=["community", "enterprise"]` we could use, I think.

Reply:
Added.